service_container/service_subscribers_locators.rst

```diff
@@ -257,7 +257,7 @@ argument of type ``service_locator``:
         services:
             App\CommandBus:
                 arguments:
-                    !service_locator
+                    !service_locator:
```

Comment:
Im not sure about this one. Im getting invalid syntax both before and after the patch..

Reply:
It should probably be
```yaml
!service_locator |
  App\FooCommand: ...
```